Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -138,15 +138,23 @@ namespace Terrain
         return false;
     }
 
-    void TerrainPhysicsColliderComponent::NotifyListenersOfHeightfieldDataChange()
+    void TerrainPhysicsColliderComponent::NotifyListenersOfHeightfieldDataChange(const AZ::Aabb* dirtyRegion,
+        const Physics::HeightfieldProviderNotifications::HeightfieldChangeMask* heightfieldChangeMask)
```

Comment:
Same here, if the only reason for pointers is for backwards compatibility, I'd pass them by reference/value instead.

Reply:
Yeah, no need for it to be a pointer indeed